cli/cli.py

```diff
@@ -46,6 +49,16 @@ def format_help(self, ctx, formatter):
         self.format_options(ctx, formatter)
         self.format_epilog(ctx, formatter)
 
+    def recommend_version_upgrade(self):
+        latest_version = package.get_latest_package_version()
+        current_version = package.get_package_version()
+        if current_version != latest_version:
```

Comment:
I think that making it smaller is better :)
While developing our version could be higher